### DISCUSSION TITLE
batch video export?

### DISCUSSION THREAD

**Discussion by dnews13** [VioletteChiara/AnimalTA#14] [Q&A] [answered]
Is there any way to export videos in batches, especially when they will all be exported in the same way (tracked and smoothed, max tail length, etc)?

**Comment by VioletteChiara** [comment]
Hi! I am afraid this option is not available. I will try to add it in the future.

**Reply by VioletteChiara** [reply, marked as answer]
This is now available (update v3.2.2)! In the export pannel, a new button "export muktpile videos" will now be available.
